### PR TITLE
Installation steps for Ubuntu 14.04 added

### PR DESCRIPTION
**Pull request by ruzickap**

### CODE REVIEW

File: LINUXCOMPILE.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ### GNU Compiler
 ```
     # Ubuntu / Debian
-    sudo apt-get install libmicrohttpd-dev libssl-dev cmake build-essential libhwloc-dev
+    sudo apt install libmicrohttpd-dev libssl-dev cmake build-essential libhwloc-dev
```
Comment (psychocrypt) [inline LINUXCOMPILE.md]: Why do you removed `apt-get` with `apt` is this equivalent?

Comment (ruzickap) [inline LINUXCOMPILE.md]: "apt" is preferred over "apt-get" for new users. 
Both are doing the same, but "apt" contains features from few different commands like apt-cache+apt-get (https://www.maketecheasier.com/apt-vs-apt-get-ubuntu/)

Comment (fireice-uk) [inline LINUXCOMPILE.md]: Thanks for that. I thought that `apt` is just an alias for `apt-get`.

Comment (ruzickap) [inline LINUXCOMPILE.md]: No problem... ```apt``` and ```apt-get``` are different pieces of software (but doing almost the same things)...

```
# ls -la `which apt`
-rwxr-xr-x 1 root root 14376 Mar  8 13:48 /usr/bin/apt

# ls -la `which apt-get`
-rwxr-xr-x 1 root root 43128 Mar  8 13:48 /usr/bin/apt-get
```

File: LINUXCOMPILE.md [hunk 2]
```diff
@@ -23,6 +23,18 @@
     sudo scl enable devtoolset-4 bash
     cmake3 .
     make install
+
+    # Ubuntu 14.04
+    sudo add-apt-repository ppa:ubuntu-toolchain-r/test
+    sudo apt update
+    sudo apt install gcc-5 g++-5 make
```
Comment (ruzickap) [inline LINUXCOMPILE.md]: I do have it on my 14.04 machines:
```
# grep PRETTY_NAME /etc/os-release
PRETTY_NAME="Ubuntu 14.04.5 LTS"

# apt --version
apt 1.0.1ubuntu2 for amd64 compiled on Dec  8 2016 16:23:37
...
```

**Comment by fireice-uk** [comment]
Ok. That clears up all the concerns I think. Merging.

**Pull request merged**